frontend/packages/ceph-storage-plugin/src/models.ts

```diff
@@ -147,3 +147,16 @@ export const StorageSystemModel: K8sKind = {
   kind: 'StorageSystem',
   crd: true,
 };
+
+// This model is used by HorizontalNav to limit the exposure of tabs to ODF Dashboard
+export const StorageSystemMockModel: K8sKind = {
```

Comment:
instead of StorageSystemMockModel can we use something prefix/suffix with name dashboard? like OcsDhasboardModel (or) OdfDashbaordModel (or) DahsboardHelperModel, etc..

Reply:
name mock not looks good

Reply:
I don't think that makes sense. This model is actually a mock of StorageSystem and doesn't point a real CRD. Hence we should keep it as mock.